tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "src/**/*.ts"
   ],
   "exclude": [
+    "src/**/*.spec.ts",
```

Comment:
is it necessary?

Reply:
Not really. But the tests are running with ts-node, so you don't realy need the spec.js files anyway.